iroha/src/account.rs

```diff
@@ -32,6 +32,19 @@ impl Account {
         }
     }
 
+    /// Constructor of the detached `Account` entity without signatories.
```

Comment:
Let's switch it with `new` so `new` will have no signatories and `signed` constructor will have one signatory.

Reply:
@humb1t ok.